cmd/aws-actuator/main.go

```diff
@@ -294,6 +294,12 @@ func bootstrapCommand() *cobra.Command {
 				}
 			}
 
+			machinePrefix := cmd.Flag("machine-prefix").Value.String()
```

Comment:
Can we use $USER if it is undefined for the moment? If you don't set this flag we're back to everything being named the same.

Reply:
Oh, this is merged...

Reply:
The `--machine-prefix` defaults to the user, see https://github.com/openshift/cluster-api-provider-aws/pull/31/files#diff-5c0fa078415d2822234c56da80b46dd6R336